Source/WebCore/Modules/mediasource/SourceBuffer.cpp

```diff
@@ -773,14 +789,18 @@ void SourceBuffer::setActive(bool active)
 
     m_active = active;
     m_private->setActive(active);
-    if (!isRemoved())
+
+    if (!!m_source && !isRemoved())
         m_source->sourceBufferDidChangeActiveState(*this, active);
 }
 
 Ref<MediaPromise> SourceBuffer::sourceBufferPrivateDidReceiveInitializationSegment(SourceBufferPrivateClient::InitializationSegment&& segment)
 {
     ALWAYS_LOG(LOGIDENTIFIER);
 
+    if (!m_source)
```

Comment:
This can't happen, so an assert would be sufficient. but I guess better safe than sorry